src/main/java/boot/MyMetaModel.java

```diff
@@ -0,0 +1,41 @@
+package boot;
+
+import jdbc.JdbcTemplate;
+import persistence.entity.*;
+
+import java.util.Map;
+import java.util.stream.Collectors;
+
+public class MyMetaModel implements MetaModel {
+    private final Map<Class<?>, EntityMeta<?>> models;
+    private final Map<Class<?>, EntityPersister<?>> persisters;
+    private final Map<Class<?>, EntityLoader<?>> loaders;
+
+    public MyMetaModel(JdbcTemplate jdbcTemplate) {
+        this.models = EntityBinder.bind();
+        this.persisters = models.keySet()
+                .stream()
+                .collect(Collectors.toMap(clazz -> clazz, clazz -> new MyEntityPersister<>(jdbcTemplate, EntityMeta.from(clazz))));
+        this.loaders = models.keySet()
+                .stream()
+                .collect(Collectors.toMap(clazz -> clazz, clazz -> new MyEntityLoader<>(jdbcTemplate, EntityMeta.from(clazz))));
+    }
```

Comment:
```suggestion
    public MyMetaModel(JdbcTemplate jdbcTemplate) {
        this.models = EntityBinder.bind();
        this.persisters = models.keySet()
                .stream()
                .collect(Collectors.toMap(clazz -> clazz, clazz -> new MyEntityPersister<>(jdbcTemplate, models.get(clazz))));
        this.loaders = models.keySet()
                .stream()
                .collect(Collectors.toMap(clazz -> clazz, clazz -> new MyEntityLoader<>(jdbcTemplate, models.get(clazz))));
    }
```
사소하지만 persisters 와 loaders 를 생성할때도 이미 만들어져있는 models 를 사용해도 좋을 것 같아요 😃